cli.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'thor'
+require 'exif'
+
+class CLI < Thor
+  desc "scan", "Recursively reads all of the images from the supplied directory"
+  option :path, :type => :string, :required => false, :desc => "Optional parameter that allows any other directory to be passed in."
```

Comment:
```suggestion
  option :path, type: :string, required: false, desc: "Optional parameter that allows any other directory to be passed in."
```

this is a shorthand syntax for hashes when the key is a symbol. `{ type: :string }` is the same as `{ :type => :string }`, but we usually prefer the first (shorter) syntax

Reply:
I like it. Updated